src/main/java/online/book/store/model/User.java

```diff
@@ -31,4 +44,46 @@ public class User {
     private String shippingAddress;
     @Column(nullable = false)
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.EAGER)//TODO:EAGER is ok?
```

Comment:
I think yes, because with FetchType.LAZY i have LazyInitializationException :)

Reply:
Ok thanks)
It just seems that there are other ways to solve the LazyInitExc here, but now this is the only option that I came up with)